tests/grpc/test_grpc_client.py

```diff
@@ -0,0 +1,84 @@
+""" Testing the sync version of the client stubs.
+
+This is not testing the lower level grpc calls, but rather the generated client stubs.
+So instead of creating a real service and a real grpc channel,
+we are going to mock the channel and simply test the client.
+
+If we wanted to test without mocking we would need to use all the machinery here:
+https://github.com/grpc/grpc/blob/master/src/python/grpcio_tests/tests/testing/_client_test.py
```

Comment:
Or I could spawn an actual server. I would think that's the best testing, but it is not done so in the rest of the code (which uses the nice grpclib `ChannelFor`)